var/spack/repos/builtin/packages/ginkgo/package.py

```diff
@@ -21,6 +21,7 @@ class Ginkgo(CMakePackage, CudaPackage):
     version('master', branch='master')
     version('1.0.0', commit='4524464')  # v1.0.0
 
+    variant('cuda', default=False, description='Build with Ginkgo')
```

Comment:
Did you mean "Build with CUDA"?

Reply:
Yes definitely. Thanks for finding this.